docs/subsystems/conversion.md

```diff
@@ -9,99 +9,130 @@ administrators to follow.
-with getting Zulip itself running are out of scope here; see [the
-production installation instructions](../index.html#zulip-in-production).
-As for the import side of things, we only touch on it implicitly.  (My
-reasoning was that we *had* to get the export piece right in a timely
-fashion, even if it meant we would have to sort out some straggling
-issues on the import side later.)
+Zulip includes an export tool, `management/export.py`, which
+exports data for a single Zulip realm. `management/export.py`
+is beneficial when migrating a Zulip realm to a new server.
 
-## Exporting multiple realms' data when moving to a new server
-
-The main exporting tools in place as of summer 2016 are below:
-
-- We can export single realms (but not yet limit users within the
-  realm).
-- We can export single users (but then we get no realm-wide data in
-  the process).
-- We can run exports simultaneously (but have to navigate a bunch of
-  /tmp directories).
-
-Things that we still may need:
-- We may want to export multiple realms simultaneously.
-- We may want to export multiple single users simultaneously.
-- We may want to limit users within realm exports.
-- We may want more operational robustness/convenience while doing
-  several exports simultaneously.
-- We may want to merge multiple export files to remove duplicates.
-
-We have a few major classes of data.  They are listed below in the order
-that we process them in `do_export_realm()`:
-
-#### Public Realm Data
-
-`Realm/RealmDomain/RealmEmoji/RealmFilter/DefaultStream`.
-
-#### Cross Realm Data
-
-`Client/zerver_userprofile_cross_realm`
+This document cover the logistics a big conversion of a
+multi-realm Zulip installation, supplementing the explanation in
+`management/export.py`.
 
-This includes `Client` and three bots.
+There are many major operational aspects to doing a conversion,
+including:
 
-`Client` is unique in being a fairly core table that is not tied to
-`UserProfile` or `Realm` (unless you somewhat painfully tie it back to
-users in a bottom-up fashion though other tables).
+- Get new servers running
+- Export data from the old DB
+- Export files from Amazon S3
+- Import files into new storage
+- Import data into new DB
+- Restart new servers
+- Decommission old server
 
-#### Disjoint User Data
+This document focuses on **exporting** data from the old DB and
+Amazon S3.  For informatoin on installing or running Zulip see
+[Zulip installation instructions](../index.html#zulip-in-production).
 
-`UserProfile/UserActivity/UserActivityInterval/UserPresence`.
-
-#### Recipient Data
-
-`Recipient/Stream/Subscription/Huddle`.
-
-These tables are tied back to users, but they introduce complications
-when you try to deal with multi-user subsets.
-
-#### File-related Data
+## Exporting multiple realms' data when moving to a new server
 
-`Attachment`
+As of summer 2016, Zulip can:
+- Export single realms (but not yet limit users within the
+  realm)
```

Comment:
Each option listed with its limitations here makes it appear like we can do significantly less work than we actually can.

If we decide to keep this doc, I feel we should write the part in brackets below after this initial list of types of exports we support.

Reply:
That makes sense to me. I haven't change this, as it will take a bit more time than the other changes. If @timabbott wants to keep this doc, I can make the changes! I'll wait and see what @timabbott says.